kotlin-asyncapi-annotation/src/main/kotlin/org/openfolder/kotlinasyncapi/annotation/AsyncApiDocumentation.kt

```diff
@@ -0,0 +1,4 @@
+package org.openfolder.kotlinasyncapi.annotation
+
+@Target(AnnotationTarget.CLASS)
+annotation class AsyncApiDocumentation
```

Comment:
I would still suggest `@AsyncApiComponent` to use the common Async API terminology. It would refer to the generic component model of Async API (which contains other models like channels, messages, schemas...).

Reply:
Updated.